src/main/java/com/google/sps/servlets/ScheduledInterviewServlet.java

```diff
@@ -430,8 +484,20 @@ private static String formatPositionString(String str) {
     String splitString[] = str.split("_", 0);
     String formattedPositionString = "";
     for (String s : splitString) {
-      formattedPositionString += s.substring(0, 1) + s.substring(1).toLowerCase() + " ";
+      formattedPositionString += s.substring(0, 1) + s.substring(1).toLowerCase();
     }
     return formattedPositionString.trim();
   }
+
+  private ScheduledInterview updatedMeetLinkField(
+      String meetLink, ScheduledInterview previousInterviewObject) {
+    return ScheduledInterview.create(
+        previousInterviewObject.id(),
+        previousInterviewObject.when(),
+        previousInterviewObject.interviewerId(),
+        previousInterviewObject.intervieweeId(),
+        meetLink,
+        previousInterviewObject.position(),
+        previousInterviewObject.shadowId());
+  }
```

Comment:
This shouldn't be necessary anymore with the withMeetLink function.

Reply:
Done